src/test/sioc.c

```diff
@@ -106,7 +106,7 @@ static int generic_request_by_name(int sockfd, struct ifreq* req, int nr,
   ret = ioctl(sockfd, nr, req);
   VERIFY_GUARD(req);
   atomic_printf("%s(ret:%d): %s ", nr_str, ret, req->ifr_name);
-  if (ret < 0 && (nr == SIOCGIFADDR || nr == SIOCGIFDSTADDR)) {
+  if (ret < 0 && (nr == SIOCGIFADDR || nr == SIOCGIFDSTADDR || nr == SIOCGIFBRDADDR || nr == SIOCGIFNETMASK)) {
```

Comment:
Why don't we just allow this on all ioctls? No harm.

Reply:
If I understand it right, checking the errors that strict as it was is less a test for rr as more for the kernel.
So I am all open for that, I will prepare a new version.

Reply:
I pushed a new version. And got a successful run at amdci1 [here](https://buildkite.com/julialang/rr/builds/1148).